prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-page.component.ts

```diff
@@ -104,10 +116,12 @@ export class RegulatoryPageComponent extends AbstractEnrolmentPage implements On
   }
 
   protected afterSubmitIsSuccessful(): void {
+    const collegeCertifications = this.formState.collegeCertifications;
+    const formDeviceProviderIdentifier = this.formState.deviceProviderIdentifier.value;
```

Comment:
Don't need to prefix with form.  It's a scoped variable and how you access it makes context explicit.  You can also make this easier to scan since by doing something like:

```js
const isDeviceProviderWithNoIdentifier = this.isDeviceProvider && !this.formState.deviceProviderIdentifier.value;
const nextRoutePath = (!collegeCertifications.length || isDeviceProviderWithNoIdentifier)
  ? ...
```

Reply:
Changed